PeerioTechnologies/peerio-desktop
# fix: isDevEnv on Linux

Recently require() was changed to import, which caused the fix for
process.env to not work, since imports are loaded statically before
executing code.

#### Relevant info and issue/PR links



#### Testing instructions

In Linux production build in console, type `require('./helpers/is-dev-env')`.

Check that output is:

`{default: false, __esModule: true}`


----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
